premailer/premailer.py

```diff
@@ -161,7 +106,8 @@ def __init__(self, html, base_url=None,
                  method="html",
                  base_path=None,
                  disable_basic_attributes=None,
-                 disable_validation=False):
+                 disable_validation=False,
+                 cache_css_parsing=True):
```

Comment:
You did a nice test that checked that it's possible to switch off the cache. Let's leave it on.